.buildkite/pipeline.yml

```diff
@@ -2,11 +2,11 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/a8c-ci-toolkit#3.1.0
+    - $CI_TOOLKIT_PLUGIN
   # Common environment values to use with the `env` key.
   - &common_env
     # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
-    IMAGE_ID: xcode-15.1
+    IMAGE_ID: $IMAGE_ID
```

Comment:
…and then at that point we could also consider moving this env definition at top-level to have Buildkite apply it to all steps (like how it's already done in `.buildkite/release-builds.yml`) and remove the need for repeating `env: *common_env` in each individual step (even if you have some steps that don't run on a Mac agent and don't need that `IMAGE_ID` env, it doesn't hurt the step if it's present anyway, so…).

With that you'd be able to get rid of the `&common_env` anchor as well… and thus be able to remove the whole `common_params` section altogether to simplify the pipeline even further 🙂

Reply:
>  (even if you have some steps that don't run on a Mac agent and don't need that IMAGE_ID env, it doesn't hurt the step if it's present anyway, so…).

I hadn't considered this point. Yes, makes lots of sense as a tidy up.